Makefile

```diff
@@ -44,5 +44,7 @@ clean-aci:
 integration: build-test-image push-test-image test-aci
 
 integration-local: build-test-image
-	@docker run --env-file .env --rm --entrypoint go test -run TestConnectionString -v ./integration ${IMAGE}
+	@docker-compose --env-file .env up
```

Comment:
added a docker-compose.yaml file that sets up jaeger and runs integration tests locally (targetting the servicebus namespace as configured in .env)